Esri/arcgis-maps-sdk-kotlin-samples
# New Indoor positioning Sample

## Description

PR to add a new Kotlin sample _"Show Device Location Using Indoor Positioning"_ in `Maps and Scenes` category.

## Links and Data

Sample Epic: `runtime/kotlin/issues/2011`

## What To Review

-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files


## How to Test

Replace line 107-108 from MainActivity.kt with this section of code, to test the sample using IPS webmap from redlands Building M. You need to be in Esri building M, to test this.

```
val itemUrl =
    "https://runtimecoretest.maps.arcgis.com/home/item.html?id=41fd5a7ee7044a40ad73f7f4a79a5a25"
val authenticator = ArcGISAuthenticationChallengeHandler {
    ArcGISAuthenticationChallengeResponse.ContinueWithCredential(
        TokenCredential.create(
            itemUrl,
            "username",
            "password",
            0
        ).getOrThrow()
    )
}
ArcGISEnvironment.authenticationManager.arcGISAuthenticationChallengeHandler = authenticator
mapView.map = ArcGISMap(itemUrl)
```

please see the linked kotlin issue for username and password
